shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -100,21 +192,24 @@ private void load(AlluxioURI filePath, int replication)
         load(newPath, replication);
       }
     } else {
-      Thread thread = CommonUtils.createProgressThread(System.out);
-      thread.start();
-      try {
-        JobGrpcClientUtils.run(new LoadConfig(filePath.getPath(), replication), 3,
-            mFsContext.getPathConf(filePath));
-      } finally {
-        thread.interrupt();
+      if (status.getInAlluxioPercentage() == 100) {
+        // The file has already been fully loaded into Alluxio.
+        System.out.println(filePath + " already in Alluxio fully");
```

Comment:
` is already fully loaded into Alluxio`?

Reply:
Is there any more reasonable condition, please tell me.

Reply:
Calvin is simply suggesting
`System.out.println(filePath + " is already fully loaded in Alluxio");`